icons/activity.tsx

```diff
@@ -34,10 +39,11 @@ const variants: Variants = {
 
 const ActivityIcon = forwardRef<
   ActivityIconHandle,
-  HTMLAttributes<HTMLDivElement>
+  ActivityIconProps
 >(({ onMouseEnter, onMouseLeave, ...props }, ref) => {
```

Comment:
can you update to this please 
```suggestion
>(({ onMouseEnter, onMouseLeave, className, size = 28, ...props }, ref) => {
```